### PR TITLE
Test-Json: document SchemaFile parameter

### PR DESCRIPTION
**Pull request by beatcracker**
# PR Summary

Document `SchemaFile` parameter for the `Test-Json` cmdlet.
See https://github.com/PowerShell/PowerShell/pull/11934 for details

Fixes #6134 

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7.1 preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual content**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles
- [ ] Community content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/7.1/Microsoft.PowerShell.Utility/Test-Json.md [hunk 1]
```diff
@@ -136,7 +136,27 @@ For more information, see [JSON Schema](https://json-schema.org/).
...
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -SchemaFile
+
+Specifies a Schema file to validate the JSON input against. If passed `Test-Json` will validate that
+the Json input conforms to the spec specified by the **SchemaFile** parameter and return `$True`
+only if the input conforms to the provided Schema.
```
Comment (sdwheeler) [inline reference/7.1/Microsoft.PowerShell.Utility/Test-Json.md:153]:
```suggestion
Specifies a schema file used to validate the JSON input. When used, the `Test-Json` returns `$True`
only if the JSON input conforms to the Schema defined in the file specified by the **SchemaFile**
parameter.
```

**Comment by beatcracker** [comment]
@sdwheeler I've added an example. Not sure if that what you had in mind, but I hope that we can work from this.

**Comment by sdwheeler** [comment]
@beatcracker Thanks for the example. I reformatted it to fix the build error. I see that this is still `[WIP]`. Do you have anything else to add?

**Comment by beatcracker** [comment]
@sdwheeler Thank you! I just realized that I've botched a second JSON in example, so here is the fix. If it builds properly and you're ok with it, I'll remove the `[WIP]` badge .

**Comment by beatcracker** [comment]
@sdwheeler Done!

**Comment by sdwheeler** [comment]
Thanks for the contribution @beatcracker.

**Pull request merged**